tools/overlay_images/overlay_images.xml

```diff
@@ -54,7 +59,12 @@
         </conditional>
     </inputs>
     <outputs>
-        <data format="tiff" name="out" from_work_dir="output.tif" />
+        <data format="tiff" name="out" from_work_dir="output.tif">
+            <filter>method_option['method'] != 'seg_contour'</filter>
+        </data>
+        <data format="png" name="out" from_work_dir="output.png">
+            <filter>method_option['method'] == 'seg_contour'</filter>
+        </data>
     </outputs>
     <tests>
         <test>

```

Comment:
It would be nice if you could add `expect_num_outputs` https://docs.galaxyproject.org/en/latest/dev/schema.html#id94 to the tests. This is mainly to check that the filters are working on the outputs.